tests/setuptab.test.js

```diff
@@ -340,18 +349,27 @@ describe('UI spec functionality', () => {
       args: {
         controller: {
           name: 'afoo',
-          type: 'csv',
-          ui_control: ['arg2'],
+          type: 'csv'

```

Comment:
A few lines above where I can't leave a comment, there's a code comment that I think we can get rid of now.